docs/docs/virtual-hosts/index.md

```diff
@@ -14,4 +12,6 @@ links:
         - virtual-hosts/edit-virtual-host
     featured:
 required_permissions:
+    previous:
+    next:
 ---
```

Comment:
`previous` and `next` should not be nested under `required_permissions`.

Reply:
Fixed!